.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Question: are there any downdisdes to having `.prettierrc` be on GitHub? Should it be added to `.gitignore`? Why/why not?

Reply:
it shouldn't i thought it was in .gitignore .. Also no downside